samples/sid_end_device/sysbuild/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
-
+CONFIG_BOOT_MAX_IMG_SECTORS=256
```

Comment:
Is't it already in main prj file? https://github.com/nrfconnect/sdk-sidewalk/blob/7d52d83fc8993ed6a748eb9d20d8e96dc708a0a7/samples/sid_end_device/sysbuild/mcuboot/prj.conf#L17

Reply:
I can remove it, but the content of this file is the same as in mcuboot default configuration file for this target